effects/drugs_effect.py

```diff
@@ -0,0 +1,43 @@
+import message_text_config as msg
+import utils
+from citizens.citizen import Citizen
+from citizens.player import Player
+from game import Game
+
+from effects.effect import Effect
+
+
+class DrugsEffect(Effect):
```

Comment:
I'd prefere other effect name:
DrugEffect name is misleading and does not match original idea.

Purposal:
JunkyEffect
or FreakEffect
or smthing other.

Reply:
FreakEffect the best in my oppinion

Reply:
Done